tutordistro/distro/extra_commands/infrastructure/tutor_commands.py

```diff
@@ -69,17 +69,24 @@ def run_command(self, command: str):
             command (str): Tutor command.
         """
         try:
-            process = subprocess.run(
+            print(f'Running "{command}"')
+
+            with subprocess.Popen(
                 command,
                 shell=True,
-                check=True,
-                capture_output=True,
                 executable="/bin/bash",
-            )
-            # This print is left on purpose to show the command output
-            print(process.stdout.decode())
```

Comment:
Was this print statement not working correctly and showing the output? If it was, I suggest keeping it. Displaying the output of commands is generally a good practice, especially for debugging purposes and understanding the behavior of the code/program.

Reply:
I removed that `print` because it did not show a useful output with the commands I tested, nevertheless, thinking about it... maybe some no-tested commands would have a needed output, so I added it again

Reply:
Hmm, maybe we can test this more thoroughly to see if it’s really showing useful output or figure out how to make it show useful output. However, we can address this in another issue to avoid blocking this PR from closing. Thanks